lib/language.js

```diff
@@ -0,0 +1,855 @@
+      cs: 'Nízká glykémie'
+      }
+    ,'Urgent Low Alarm' : {   
+      cs: 'Urgentní nízká glykémie'
+      }
+    ,'Stale Data: Warn' : {   
+      cs: 'Zastaralá data'
+      }
+    ,'Stale Data: Urgent' : {   
+      cs: 'Zastaralá data urgentní'
+      }
+    ,'mins' : {   
+      cs: 'min'
+      }
+    ,'Night Mode' : {   
+      cs: 'Noční mód'
+      }
+    ,'When enabled the page will be dimmed from 10pm - 6am.' : {   
+      cs: 'Když je povoleno, obrazovka je ztlumena 22:00 - 6:00'
+      }
+    ,'Enable' : {   
+      cs: 'Povoleno'
+      }
+    ,'Settings' : {   
+      cs: 'Nastavení'
+      }
+    ,'Show Raw BG Data' : {   
+      cs: 'Zobraz RAW data'
+      }
+    ,'Never' : {   
+      cs: 'Nikdy'
+      }
+    ,'Always' : {   
+      cs: 'Vždy'
+      }
+    ,'When there is noise' : {   
+      cs: 'Při šumu'
+      }
+    ,'When enabled small white dots will be disaplyed for raw BG data' : {   
+      cs: 'Když je povoleno, malé tečky budou zobrazeny pro RAW data'
+      }
+    ,'Custom Title' : {   
+      cs: 'Vlastní název stránky'
+      }
+    ,'Theme' : {   
+      cs: 'Téma'
+      }
+    ,'Default' : {   
+      cs: 'Výchozí'
+      }
+    ,'Colors' : {   
+      cs: 'Barevné'
+      }
+    ,'Reset, and use defaults' : {   
+      cs: 'Vymaž a nastav výchozí hodnoty'
+      }
+    ,'Calibrations' : {   
+      cs: 'Kalibrace'
+      }
+    ,'Alarm Test / Smartphone Enable' : {   
+      cs: 'Test alarmu'
+      }
+    ,'Bolus Wizard' : {   
+      cs: 'Bolusový kalkulátor'
+      }
+    ,'in the future' : {   
+      cs: 'v budoucnosti'
+      }
+    ,'time ago' : {   
+      cs: 'min zpět'
+      }
+    ,'hr ago' : {   
+      cs: 'hod zpět'
+      }
+    ,'hrs ago' : {   
+      cs: 'hod zpět'
+      }
+    ,'min ago' : {   
+      cs: 'min zpět'
+      }
+    ,'mins ago' : {   
+      cs: 'min zpět'
+      }
+    ,'day ago' : {   
+      cs: 'den zpět'
+      }
+    ,'days ago' : {   
+      cs: 'dnů zpět'
+      }
+    ,'long ago' : {   
+      cs: 'dlouho zpět'
+      }
+    ,'Clean' : {   
+      cs: 'Čistý'
+      }
+    ,'Light' : {   
+      cs: 'Lehký'
+      }
+    ,'Medium' : {   
+      cs: 'Střední'
+      }
+    ,'Heavy' : {   
+      cs: 'Velký'
+      }
+    ,'Treatment type' : {   
+      cs: 'Typ ošetření'
+      }
+    ,'Raw BG' : {   
+      cs: 'Glykémie z RAW dat'
+      }
+    ,'Device' : {   
+      cs: 'Zařízení'
+      }
+    ,'Noise' : {   
+      cs: 'Šum'
+      }
+    ,'Calibration' : {   
+      cs: 'Kalibrace'
+      }
+    ,'1' : {   
+      cs: '1'
+      }
+ 
+ };
+  
+ language.translate = function translate(text) {
+    if (translations[text] && translations[text][lang])
+      return translations[text][lang];
+    return text;
+  }
+    
+  if (typeof window !== 'undefined') {
+    // do translation of static text on load
+    $('.translate').each(function (s) {
+      $(this).text(language.translate($(this).text()));
+    });
+    $('.titletranslate').each(function (s) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['s' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/797611871/issues/source?bid=2452379&fileBranchId=2723722#l846)